satpy/readers/vii_l1b_nc.py

```diff
@@ -141,5 +142,5 @@ def _calibrate_refl(radiance, angle_factor, isi):
             numpy ndarray: array containing the calibrated reflectance values.
 
         """
-        refl_values = (np.pi / isi) * angle_factor * radiance
+        refl_values = (np.pi / isi) * angle_factor * radiance * 100.0  # should be in %?
```

Comment:
Yes, we like our reflectance to be `%`. Please remove that comment.

Reply:
By the way, discussion with devs is best held here in the PR comments, rather than in the code itself :)

Reply:
Fine I will remove the comment. Its since been clarified that it should be %